app/javascript/features/lobby/lobbySlice.js

```diff
@@ -67,6 +67,18 @@ export const leaveBattle = createAsyncThunk(
   }
 )
 
+export const kickPlayer = createAsyncThunk(
+  "lobby/kickPlayer",
+  async ({ battleId, playerId, kickPlayerId }, { _, rejectWithValue }) => {
+    try {
+      const response = await axios.post("/battles/player/kick", { battleId, playerId, kickPlayerId })
```

Comment:
The API only cares about `battleId` and `kickPlayerId`, can remove `playerId` here

Reply:
is the current authed user passed via the auth then?

Reply:
We have [request helpers](https://github.com/mma127/omg_rails/blob/main/app/api/omg/helpers/request_helpers.rb) that use devise to read the session cookie and authenticate the user from that.

Not all API endpoints are authenticated, but we can see that the one you created is because in the request namespace we run a before action to authenticate: https://github.com/mma127/omg_rails/pull/205/files#diff-ccae38c37a67e9b4072493020110ea78ee9d537ed3a815cd880a7467c658960eL41-L44